opentimelineio/schema/timeline.py

```diff
@@ -68,6 +68,11 @@ def __init__(
         dict,
         "Metadata dictionary."
     )
+    global_start_time = core.serializable_field(
+        "global_start_time",
+        opentime.RationalTime,
+        doc="Global starting time value and rate of the timeline."
```

Comment:
Hmm.. interesting that this also implies the edit rate of the timeline.

Reply:
Yeah.  What do you think? Really it gives the timecode of the first time in the timeline.

Reply:
...but that implies the rate

Reply:
I don't mind that so much in principle, but given that there's a default rate of 24 I think it will be a problem in practice.

Reply:
Is there a clean way to remove the default rate?

Reply:
I'm not sure that there is a clean way to remove the default.  `RationalTime` objects are immutable, so it can be `None` or not set AFAIK.  we should discuss further.